aiohttp/errors.py

```diff
@@ -170,3 +170,13 @@ class LineLimitExceededParserError(ParserError):
     def __init__(self, msg, limit):
         super().__init__(msg)
         self.limit = limit
+
+
+class FingerprintMismatch(Exception):
+    """SSL certificate does not match expected fingerprint."""
+
+    def __init__(self, expected, got, host, port):
+        self.expected = expected
+        self.got = got
+        self.host = host
+        self.port = port
```

Comment:
Please add `__repr__` also: it's required if you add new parameters to ctor.
`repr(exc)` should display all important error about exception.